src/vmq_reg.erl

```diff
@@ -332,7 +328,8 @@ deliver_retained({MP, _} = SubscriberId, Topic, QoS) ->
       end, ok, MP, Topic).
 
 subscriptions_for_subscriber_id(SubscriberId) ->
-    plumtree_metadata:get(?SUBSCRIBER_DB, SubscriberId, [{default, []}]).
```

Comment:
the option `[{default, []}]` is passed to tell plumtree to return the `[]`  instead of undefined if no value can be found for the given key. We handle this manually in `vmq_subscriber_db`.